tsfresh/feature_extraction/extraction.py

```diff
@@ -5,8 +5,6 @@
 This module contains the main function to interact with tsfresh: extract features
 """
 
-from __future__ import absolute_import, division
-
 import logging
 import sys
 import warnings
```

Comment:
There is still an import of "six" and a check for the python version in this file. This should also be removed.

Reply:
Done.